docs/samples/Microsoft.ML.Samples/Dynamic/PermutationFeatureImportance/PFIHelper.cs

```diff
@@ -3,7 +3,7 @@
 using Microsoft.Data.DataView;
 using Microsoft.ML.Trainers;
 using Microsoft.ML.SamplesUtils;
-using Microsoft.ML.Trainers.HalLearners;
+using Microsoft.ML.Trainers.Mkl;
```

Comment:
>Microsoft.ML.Trainers.Mkl [](start = 6, length = 25)

in the last API review we decided to not have sub namespaces in Microsoft.ML.Trainers. (except FastTree). 
So Microsoft.ML.Trainers.HalLearners; -> Microsoft.ML.Trainers

Reply:
Agreed. I don’t think there are enough types to justify a new namespace.

Reply:
Ok, I’ll revert this change. Thanksz